docs/service-discovery/dns.md

```diff
@@ -0,0 +1,44 @@
+# DNS Service Discovery
+
+DNS is a name resolution protocol used to determine IP addresses for hostnames.
+That makes it a natural fit for service discovery. Among others, Consul and AWS Cloud Map provide DNS APIs for service discovery.
+
+This page explains how Stork can use DNS to handle the service discovery.
+
+## Dependency
+
+First, you need to add the Stork Consul Service Discovery provider:
+
+```xml
+<dependency>
+    <groupId>io.smallrye.stork</groupId>
+    <artifactId>stork-service-discovery-dns</artifactId>
+    <version>{{version.current}}</version>
+</dependency>
+```
+
+## Configuration
+
+At the moment DNS service discovery in Stork does not support reading the port of the service from DNS, hence on top of specifying the service discovery `type`, you need to specify `port` in your configuration. 
+Additionally, you would usually want to specify the DNS server, or servers, to use for the discovery. All in all, your configuration could look as follows:
+
+
+=== "stork standalone"
+    ```properties
+    stork.my-service.service-discovery.type=dns
+
+    # optional dns servers:
+    stork.my-service.service-discovery.dns-servers=my-dns-server:8221,my-dns-server2
```

Comment:
Is the 8221 the port of which you are talking about in line 22?
If yes:
1- why there is no port for the `my-dns-server2` config?
2- According to this sentence `hence on top of specifying the service discovery `type`, you need to specify `port` in your configuration` and the code of the DnsServiceDiscovery.getPort() method I would expect a `stork.my-service.service-discovery.dns-server-port` or something like that.

Reply:
53 is the default port. no need to specify the port if the default is used.

It's not the port I meant in the doc. I missed it :(